test/functional/tests/cache_ops/test_multilevel_cache.py

```diff
@@ -18,11 +18,11 @@ def test_remove_multilevel_core():
     """
     title: Test of the ability to remove a core used in a multilevel cache.
     description: |
-        Negative test if OpenCAS does not allow to remove a core when the related exported object
+        Negative test if it is not allowed to remove a core when the related exported object
```

Comment:
```suggestion
        Negative test for removing a core when the related exported object
```
Quite acrobatic syntax ;)